community/contributing/bisect_workflow.rst

```diff
@@ -0,0 +1,136 @@
+.. _doc_bisect_workflow:
+
+Bisect workflow
+===============
+
+.. highlight:: shell
+
+Bisecting is a way to find regressions in software. After reporting a bug on the
+`Godot repository on GitHub <https://github.com/godotengine/godot>`__, you may
```

Comment:
rST external hyperlinks take only one trailing underscore, not two (same for all links).